canonical-websites/design.ubuntu.com
# update the header

## Done

* removed search
* updated the html to latest vanilla

## QA

* download and `run serve`
* open any [page](http://0.0.0.0:8011/)
* see that the header looks as it should based on [vanilla](https://docs.vanillaframework.io/en/patterns/navigation) in all browser sizes, but is orange

## Screenshots

![image](https://user-images.githubusercontent.com/441217/30385204-5d11f17a-989e-11e7-9b2a-440ead3bc198.png)

## Issues

Fixes #52
